setup.py

```diff
@@ -54,9 +54,9 @@ def read(fname):
       },
       install_requires=[
           "mbed-ls>=1.5.1,==1.*",
-          "six",
-          "pyserial",
-          "pyOCD!=0.13.0"
+          "six==1.*",
+          "pyserial==3.*",
+          "pyOCD>=0.13.1,<0.14.0"
```

Comment:
is there coming breaking changes to pyocd?

Reply:
In Mbed OS we currently pin the pyocd version because it is not pinned here and can cause breakages. I'm now being asked to update the version within Mbed OS to version 0.16.x due to new target support: https://github.com/ARMmbed/mbed-os/pull/9481#issuecomment-461118156

Do you actively use pyocd to flash? Would it be possible to make it an optional dependency like we currently do in mbed-os-tools:
- https://github.com/ARMmbed/mbed-os-tools/blob/master/setup.py#L60-L62
- https://github.com/ARMmbed/mbed-os-tools/blob/a6f73ab6353ee2d909518c99a159deaceee494d1/src/mbed_os_tools/test/host_tests_plugins/module_copy_pyocd.py#L51-L56

Reply:
we are not using pyocd actively yet, in fact pyocd integration is broken in master 😄. but we have near future plans to start using pyocd also in CI so this PR is just fine for now if its okay for you. We have to think about if it could be optionally but lets pin it first..

Reply:
If pyocd is broken in master, do you mind if I either:

- Remove the pyocd implementation
- Make the imports conditional? So if Pyocd isn't present, it'll just print an error saying you need to install it? (we do this in mbed-os-tools, link above)

Reply:
{quote}
* Remove the pyocd implementation
{quote}

This is okay with me. According to https://github.com/ARMmbed/isgtesting-arch/pull/24 we will be working on **pyOCD** integration on **mbed-flasher** branch.